apps/backoffice-v2/src/lib/blocks/components/TableCell/TableCell.tsx

```diff
@@ -10,98 +10,96 @@ import {
   TableRow,
 } from '../../../../common/components/atoms/Table';
 import { ITableCellProps } from './interfaces';
-import { ctw } from '../../../../common/utils/ctw/ctw';
+import { ctw } from '@/common/utils/ctw/ctw';
 import { DefaultCell } from './DefaultCell';
 
-export const TableCell = forwardRef(
-  <TData extends RowData, TValue = unknown>(
-    { value }: ITableCellProps<TData, TValue>,
-    ref: ForwardedRef<ElementRef<typeof Table>>,
-  ) => {
-    const table = useReactTable<TData>({
-      ...value?.options,
-      data: value.data ?? [],
-      columns: value.columns ?? [],
-      getCoreRowModel: getCoreRowModel(),
-      defaultColumn: {
-        cell: DefaultCell,
-      },
-    });
+export const TableCell = <TData extends RowData, TValue = unknown>({
```

Comment:
`any` works better right?
```suggestion
export const TableCell = <TData extends RowData, TValue = any>({
```